service/src/main/kotlin/fi/espoo/evaka/invoicing/service/VoucherValueDecisionGenerator.kt

```diff
@@ -200,7 +203,7 @@ private fun generateNewValueDecisions(
                     child = voucherChild,
                     baseCoPayment = baseCoPayment,
                     siblingDiscount = price.siblingDiscountPercent(1),
-                    baseValue = baseValue,
+                    baseValue = 0,
```

Comment:
Can `baseValue` be zero for empty decisions? We cannot have any other number really, because there's no placement and thus no service need.

Reply:
Yeah, I think that's fine. It doesn't make sense either way really 😄